includes/Products/Feed.php

```diff
@@ -179,6 +187,192 @@ public function schedule_feed_generation() {
 	}
 
 
+	/**
```

Comment:
change comment on 181 that says it defaults to 15 min if we are doing it every day